### PR TITLE
#12: Added Docs

### PR DESCRIPTION
**Pull request by uglyprincess**
Proposed Changes
- Added docs on how to contribute to this project 
-This fixes issue #12 

- [x] My code follows the Mentioned Guidelines for the project
- [x] I have self-reviewed the changes made
- [x] I have made the required changes to the documentations

P.S. Please review this changes and suggest if I left anything out. Thanks!

### CODE REVIEW

**Review by Tlazypanda — COMMENTED** [review]
Hey @uglyprincess are the commands enclosed in code quotes and should we have instructions for a feature branch too?

**Comment by uglyprincess** [comment]
> Hey @uglyprincess are the commands enclosed in code quotes and should we have instructions for a feature branch too?

I'll make those changes ASAP! Could you provide a little details about the feature branch?

**Comment by Tlazypanda** [comment]
Hey @uglyprincess https://systers-portal.readthedocs.io/en/develop/develop/contributing.html this might be helpful 😅

File: How To Contribute.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+If you're unfamiliar with Git Bash or if this is your first time contributing, follow these steps to get started:
+
+1. Download git bash for your device: [Git Bash](https://git-scm.com/downloads)
+
+1. Open git bash anywhere on your local server by clicking the right mouse button. Then set up your name and email ID:
+
+	`$ git config --global user.name "<Your_Name>"`
+	`$ git config --global user.email "<Your_Email>"`
+
+3.
```
Comment (rush-tea) [inline How To Contribute.md:14]: you forgot something i guess

**Review by rush-tea — COMMENTED** [review]
Point 3 has nothing in it 😬😀

**Comment by uglyprincess** [comment]
> Point 3 has nothing in it 😬😀

I was in the middle of editing the docs, that's probably why. Next commit will fix this! 😄

**Comment by uglyprincess** [comment]
Final version of the docs committed! Please review the changes @Tlazypanda (hopefully I didn't leave anything out this time around 😝 )

**Review by Tlazypanda — APPROVED** [review]
@uglyprincess Changes look good :tada:

**Comment by Tlazypanda** [comment]
@all-contributor add @uglyprincess for docs

**Comment by Tlazypanda** [comment]
@all-contributors add @uglyprincess for docs

**Pull request merged**